mycalendar/src/webapp/scripts/mycalendar.js

```diff
@@ -0,0 +1,263 @@
+(function($) {
+
+    var currentSiteId;
+    var timeZone;
+    var eventsList = [];
+
+    $(document).ready(function () {
+
+        moment.locale(sakai.locale.userLocale);
+        
+        // data
+        currentSiteId = $(".mycalendar-data").attr("data-siteid");
+        timeZone = $(".mycalendar-data").attr("data-tz");
+        
+        //get all events for the site for this month
+        var todaysDate = moment().format('YYYY-MM-DD');
+        eventList = getEvents(todaysDate);
+
+        var init = function () {
+        
+                $(".mycalendar").datepicker({
```

Comment:
But, it's just a datepicker, right? In this case we are using the date picker as an actual calendar, with events showing as highlighted and the ability to navigate the months and days. The localised one's based on datepicker, so it should be possible to adapt the code.
@mgolden-noodle  ?